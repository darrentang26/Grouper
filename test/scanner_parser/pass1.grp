type Fruit = APPLE | BANANA | ORANGE

let Int z = 4 
and Int x = 3
and Int list xs = []
and Int list ys = 4::[1,2,3]
and Int y = 5 in
   if true then "yay" else "no" end
