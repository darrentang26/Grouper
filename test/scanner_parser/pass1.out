type Fruit = APPLE | BANANA | ORANGE
let Int z = 4 in
let Int x = 3 in
let Int list xs = [] in
let Int list ys = (4 :: (1 :: (2 :: (3 :: [])))) in
let Int y = 5 in
if true then "yay" else "no"
